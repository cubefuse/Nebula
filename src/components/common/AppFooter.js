// @flow
import React from "react";
import { Layout } from "antd";

const { Footer } = Layout;

/**
 * App Footer component.
 */
export default class AppFooter extends React.Component<{}> {
  render() {
    return (
      <Footer style={{ textAlign: "center" }}>
        Copyright Â© 2018 Cubefuse. Distributed under MIT license.
      </Footer>
    );
  }
}
