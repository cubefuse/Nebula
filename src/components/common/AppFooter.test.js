import React from "react";
import "jest-dom/extend-expect";
import { render } from "react-testing-library";
import AppFooter from "./AppFooter";

test("can render", () => {
  const node = render(<AppFooter />);
  expect(node).toMatchSnapshot();
});
