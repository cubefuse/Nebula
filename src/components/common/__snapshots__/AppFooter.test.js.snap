// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render 1`] = `
Object {
  "container": <div>
    <div
      class="ant-layout-footer"
      style="text-align: center;"
    >
      Copyright Â© 2018 Cubefuse. Distributed under MIT license.
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
